stablehlo/dialect/StablehloAttrs.td

```diff
@@ -21,8 +21,8 @@ include "mlir/IR/OpBase.td"
 include "mlir/IR/TensorEncoding.td"
 
 def StableHLO_Dim : ArrayRefParameter<"int64_t", "Dimension"> {
```

Comment:
In MHLO, we recently renamed this to `MHLO_Dims`. Since we're here, how about we rename this to `StableHLO_DIms`? This is a plural thing after all.